src/wallet/wallettool.cpp

```diff
@@ -95,6 +95,9 @@ static void WalletShowInfo(CWallet* wallet_instance)
     LOCK(wallet_instance->cs_wallet);
 
     tfm::format(std::cout, "Wallet info\n===========\n");
+    tfm::format(std::cout, "Name: %s\n", wallet_instance->GetName());
```

Comment:
The linter is complaining about trailing whitespace in the test changes, but the whitespace is needed for the test to pass. Maybe print `Name:` or `Name: ""` when `wallet_instance->GetName().empty()`

Reply:
An alternative would be to write

```
    Name: \

    Format: bdb
```

in the test

Reply:
fixed.

Reply:
still one left

`test/functional/tool_wallet.py:145:18: W291 trailing whitespace`